sdks/python/apache_beam/options/pipeline_options_test.py

```diff
@@ -623,6 +623,17 @@ def test_transform_name_mapping(self):
     mapping = options.view_as(GoogleCloudOptions).transform_name_mapping
     self.assertEqual(mapping['from'], 'to')
 
+  def test_service_options(self):
+    options = PipelineOptions(['--service_option', 'whizz=bang', '--service_option', 'beep=boop'])
+    self.assertEqual(
+        sorted(options.get_all_options()['service_options']), ['beep=boop', 'whizz=bang'])
+
+    options = PipelineOptions(['--service_options', 'whizz=bang', '--service_options', 'beep=boop'])
+    self.assertEqual(
+        sorted(options.get_all_options()['service_options']), ['beep=boop', 'whizz=bang'])
```

Comment:
are these lines exactly the same as the ones before?

Reply:
`sorted(options.get_all_options()['service_options']), ['beep=boop', 'whizz=bang'])` is the same. The `PipelineOptions` are configured once using repeated `service_option` (i.e. singular) flags, and once using `service_options` (i.e. plural).